src/assets/server/rest/v1/schemas/chargingstation/chargingstation-action-change-availability.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Change Charging Station availability (endpoint)",
```

Comment:
To rename 'Change Charging Station availability (endpoint)' -> 'Change Availability'
Whether we put Charging Station everywhere or not ;-)
To remove also '(endpoint)' string.